README.md

```diff
@@ -89,6 +90,31 @@ iex> Litmus.validate(params, schema)
 {:error, "password must be alphanumeric"}
 ```
 
+### Litmus.Type.Number
+
+The `Number` module contains options that will validate Number data types. It converts "stringified" numerical values to numbers. It supports the following options:
+  * `:min` - Specifies the minimum value of the field. Allowed values are numbers.
+  * `:max` - Specifies the maximum value of the field. Allowed values are numbers.
```

Comment:
do we need `Allowed values are numbers`?

Reply:
I know it looks weird. What would be more appropriate?